Benqi-Agents/Interest-Rates-Monitor/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "benqi-6",
+  "version": "0.0.1",
+  "description": "This agent detects QiTokens interest rates threshold excess",
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest --detectOpenHandles",
+    "format": "prettier --write \"src/**/*.ts\""
+  },
+  "dependencies": {
+    "forta-agent": "^0.0.39",
+    "@types/lru-cache": "^7.4.0"
+  },
+  "devDependencies": {
+    "@types/jest": "^27.0.1",
+    "@types/nodemon": "^1.19.0",
+    "forta-agent-tools": "^2.0.3",
+    "jest": "^27.0.6",
+    "nodemon": "^2.0.8",
```

Comment:
Thanks for the review! Fixed the devDependencies and you're certainly right about removing the `should return 8 findings` tests.

About the last test, I modified a case so both rate thresholds of a token are exceeded. Otherwise, isn't it already doing what you're suggesting? The `testQiTokens` are 4 and it creates 3 (now 4) findings out of 4 of them (one token doesn't create findings on purpose).